.github/workflows/greetings.yml

```diff
@@ -10,4 +10,4 @@ jobs:
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         issue-message: 'Thanks for your interest in Tauri!'' first issue'
-        pr-message: 'Hey there! Thanks for your contribution! :heart: Please make sure you have signed all your commits.'' first pr'
+        pr-message: 'Hey there! Thanks for your contribution! :heart: Please make sure you have signed all your commits. Signed commits mean that we can have improved level of trust that the commit was actually pushed by you.'' first pr'
```

Comment:
These signatures are lost once commits are squashed and merged, however. The only way they'll show as "Verified" once merged is if the PR author also merges their own PR (even then, the squashed commit is signed by GitHub and not the user themselves)

Reply:
Yeah - wish there was an "easy" way to do this. My hope is that by squashing the org member that made the merge uses their verification to greenlight the changes to master. Not sure if that will work tho...